.gitignore

```diff
@@ -35,3 +35,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+package-lock.json
```

Comment:
I don't think we can put it in gitignore @samurato da
It's required to run the app.

Reply:
Isn't package-lock.json generated automatically from package.json when installing dependencies??

Reply:
While package.json specifies the package versions (with ranges), package-lock.json locks them down to exact versions. If you don’t keep it, you may end up with different versions across environments. So, with package-lock.json, you get the same versions every time, without it, you might get variations depending on the specified ranges.

It also provides 
1. **Dependency Consistency**: It ensures that all developers working on a project have the exact same dependency versions installed. This prevents issues that can arise from version differences.
2. **Performance**: It speeds up the installation process since the package manager can directly use the resolved dependencies without needing to resolve them again.

Reply:
Updated gitignore